transferring-files.tex

```diff
@@ -0,0 +1,173 @@
+
+% \subsection{Transferring Files with M65Connect}
+% \index{M65Connect Application}
+% TODO: M65Connect instructions are pending an update to M65Connect that implements this procedure.
+
+\subsection{The mega65\_ftp Command-Line Tool}
+\index{mega65\_ftp command@\texttt{mega65\_ftp command}}
+
+The {\tt mega65\_ftp} command-line tool initiates a file transfer session with the MEGA65. It can run interactively in the terminal and accept multiple file transfer commands, or it can run non-interactively with those commands provided as arguments.
+
+% TODO: This may be updated in the future to allow for automatic selection of an IP address.
+
+To start an interactive file transfer session, run the {\tt mega65\_ftp} command, providing the IP address you have selected for the MEGA65 as the {\tt -i} argument.
+
+\begin{verbatim}
+% mega65_ftp -i 192.168.0.65
+\end{verbatim}
+
+The tool will upload the file transfer client, and you will see it the client running on the MEGA65. If nothing happens, press Ctrl-C (on the PC) to abort, then double-check that the MEGA65 is connected and that network listening is enabled.
+
+Once connected, the file transfer command prompt looks similar to this:
+
+\begin{verbatim}
+MEGA65 SD-Card:/>
+\end{verbatim}
+
+To end the session, use the {\tt exit} command. The tool will exit and return to the shell prompt, and the MEGA65 will reset.
+
+\begin{verbatim}
+MEGA65 SD-Card:/> exit
+%
+\end{verbatim}
+
+The following are several useful commands you can use during the file transfer session. Use the {\tt help} command to see a complete list of available commands.
+
+\begin{center}
+\begin{tabular}{|l|l|}
+\hline
+{\bf Command} & {\bf Description} \\
+\hline
+{\tt put {\it filename}} & Send a file from the PC to the MEGA65. \\
+\hline
+{\tt get {\it filename}} & Retrieve a file from the MEGA65 to the PC. \\
+\hline
+{\tt dir} & Display a directory listing of the MEGA65 SD card. \\
+\hline
+{\tt ldir} & Display a directory listing of the local current working directory. \\
+\hline
+{\tt mkdir {\it dirname}} & Create a sub-directory on the MEGA65 SD card. \\
+\hline
+{\tt cd {\it dirname}} & Change the current working directory on the MEGA65 SD card. \\
+\hline
+{\tt help} & Display a list of available commands. \\
+\hline
+{\tt exit} & End the file transfer session. \\
+\hline
+\end{tabular}
+\end{center}
+
+To invoke {\tt mega65\_ftp} commands without starting an interactive prompt, use the {\tt -c} argument once for each command:
+
+\begin{verbatim}
+% mega65_ftp -i 192.168.0.65 -c 'put mydisk.d81' -c 'exit'
+\end{verbatim}
+
+The tool will start a session, execute the commands, then terminate. Be sure to issue the {\tt exit} command as the final command to reset the MEGA65, or reset the MEGA65 manually after the file transfer has completed.
```

Comment:
Yes, this section is unfinished until M65Connect has implemented the new changes. It will tour the M65Connect UI, and recommend it as the primary way people transfer files.